scripts/update_ora2_truncated_course_ids.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+"""
+Script to fix workflows with truncated course_ids from https://github.com/Stanford-Online/edx-ora2/pull/25.
+AIClassifierSet, AIGradingWorkflow and AITrainingWorkflow excluded as they are not used by Stanford.
+"""
+from itertools import chain
+import os
+import django
+from django.db.models.functions import Length
+from openedx.core.djangoapps.monkey_patch import django_db_models_options
+
+
+def main():
+    from openassessment.assessment.models import PeerWorkflow, StaffWorkflow, StudentTrainingWorkflow
```

Comment:
Why aren't these top-level imports?

Reply:
Needs the django setup code in the if block